plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -162,3 +221,111 @@ global proc mayaUsdMenu_unloadui() {
+    int $singleSelect = `mayaUsdLayerEditorWindow -q -selectionLength $panelName` == 1;
+    int $isDirty = `mayaUsdLayerEditorWindow -q -isLayerDirty $panelName`;    
+    int $appearsMuted = `mayaUsdLayerEditorWindow -q -layerAppearsMuted $panelName`;
+    int $isSubLayer = `mayaUsdLayerEditorWindow -q -isSubLayer $panelName`;
+    int $isMuted = `mayaUsdLayerEditorWindow -q -layerIsMuted $panelName`;
+    
+    string $label;
+    int $enabled;
+
+    if (!$isSessionLayer) {
+        if ($isAnonymousLayer)
+            $label = getMayaUsdString("kMenuSaveAs");
+        else 
+            $label = getMayaUsdString("kMenuSaveEdits");
+        $enabled = $singleSelect && $needsSaving;
+        $cmd = makeCommand($panelName, "saveEdits");
+        menuItem -label $label -enable $enabled -c $cmd;
+        
+
+    }
+
+    $label = getMayaUsdString("kMenuDiscardEdits");
+    $cmd = makeCommand($panelName, "discardEdits");
+    $enabled = $isDirty;
+    menuItem -label $label -enable $enabled -c $cmd;
+    
+    menuItem -divider 1;
+
+    $label = getMayaUsdString("kMenuAddSublayer");
+    $cmd = makeCommand($panelName, "addAnonymousSublayer");
+    $enabled = $appearsMuted == 0;
+    menuItem -label $label -enable $enabled -c $cmd;
+
+    $label = getMayaUsdString("kMenuAddParentLayer");
+    $cmd = makeCommand($panelName, "addParentLayer");
+    $enabled = $isSubLayer && $appearsMuted == 0;
+    menuItem -label $label -enable $enabled -c $cmd;
+
+    $label = getMayaUsdString("kMenuLoadSublayers");
+    $cmd = makeCommand($panelName, "loadSubLayers");
+    $enabled = $singleSelect && !$appearsMuted;
+    menuItem -label $label -enable $enabled -c $cmd;
+
+    menuItem -divider 1;
+    
+    if ($isSubLayer) {
+    	if ($isMuted) 
+    		$label = getMayaUsdString("kMenuUnmute");
+    	else
+    		$label = getMayaUsdString("kMenuMute");
+	    $cmd = makeCommand($panelName, "muteLayer");
+    	$enabled = 1;    	
+	    menuItem -label $label -enable $enabled -c $cmd;
+    }
+
+    $label = getMayaUsdString("kMenuPrintToScriptEditor");
+    $cmd = makeCommand($panelName, "printLayer");
+    $enabled = 1;
+    menuItem -label $label -enable $enabled -c $cmd;
+
+    menuItem -divider 1;
+
+    $label = getMayaUsdString("kMenuSelectPrimsWithSpec");
+    $cmd = makeCommand($panelName, "selectPrimsWithSpec");
+    $enabled = 1;
+    menuItem -label $label -enable $enabled -c $cmd;
+
+    // if we add any more menus, add a divider
+    if ($isSubLayer || !$isAnonymousLayer) 
+        menuItem -divider 1;
+
+    if ($isSubLayer) {
+        $label = getMayaUsdString("kMenuRemove");
+	    $cmd = makeCommand($panelName, "removeSubLayer");
+	    $enabled = 1;
+	    menuItem -label $label -enable $enabled -c $cmd;
+	}
+
+
+    if (!$isAnonymousLayer) {
+        $label = getMayaUsdString("kMenuClear");
+	    $cmd = makeCommand($panelName, "clearLayer");
+	    $enabled = 1;
+	    menuItem -label $label -enable $enabled -c $cmd;
+	}
```

Comment:
The spacing is not all the same one these lines, and the closing } doesn't match indentation of {

Reply:
ok, this actually all lined up, but let me convert those tabs to spaces